scripts/processFactory.ts

```diff
@@ -0,0 +1,525 @@
+﻿/// <reference path="../src/compiler/sys.ts" />
```

Comment:
Let's get a description at the top of this file - while `processDiagnosticMessages` isn't documented either, it is

1. A little more obvious what it does.
2. A bad example and that's my own fault.